localization/replace/german/culture/traditions/cultural_traditions_l_german.yml

```diff
@@ -1,37 +1,37 @@
+ tradition_himalayan_settlers_name:0 "Bergsiedler"
 
- tradition_desert_ribat_name:0 "Desert Retreats"
- tradition_desert_ribat_desc:0 "This culture has a tradition of fortified desert retreats, where the pious can live in connection with the open sky and the sands. This lifestyle breeds able and pious desert warriors and attracts mystics."
+ tradition_desert_ribat_name:0 "Rückzugsorte in der Wüste"
+ tradition_desert_ribat_desc:0 "Diese Kultur hat die Tradition mit befestigten Rückzugsorten in der Wüste, wo sie in Verbindung mit dem offenen Himmel und dem Sand leben können. Dieser Lebensstil bringt fähige Wüstenkrieger hervor uns zieht Mystiker an."
  
- tradition_spartan_name:0 "Hardy"
+ tradition_spartan_name:0 "Abgehärtet"
  
- tradition_polders_name:0 "Land Reclamation"
+ tradition_polders_name:0 "Rückgewinnung von Land"
  
- culture_parameter_not_tribal_raiding_malus:0 "When returning from a successful [raid|E], non-[tribal|E] and non-[pirate|E] characters #EMP lose#! [EmptyScope.ScriptValue('not_tribal_raid_prestige_multiplier')|1] [prestige_i][prestige|E] per 1 [raid_loot|E] delivered\n#F Characters with the [GetPerk('raider_perk').GetName(GetPlayer)] [perk|E] are unaffected by this penalty#!"
+ culture_parameter_not_tribal_raiding_malus:0 "Bei einer Rückkehr von einem [raid|E] von, nicht-[tribal|E] und nicht-[pirate|E] Chrakteren #EMP verlieren#! [EmptyScope.ScriptValue('not_tribal_raid_prestige_multiplier')|1] [prestige_i][prestige|E] für jede abgelieferte [raid_loot|E] \n#F Charaktere mit den [GetPerk('raider_perk').GetName(GetPlayer)] [perk|E] bekommen keine Negativen Effekte#!"
 
- tradition_roman_legacy_name:1 "Imperial Legacy"
- tradition_roman_legacy_desc:1 "The military organization of the Cyrodiilic Empire is still in living memory within this culture. A professional army is favored over levied troops."
+ tradition_roman_legacy_name:1 "Kaiseliches Erbe"
+ tradition_roman_legacy_desc:1 "Die Militärische Organisation des Cyrodiilischen Kaiserreichs ist immer noch in dieser Kultur verankert. Eine professionelle Armee wird immer noch gegenüber erhobenen Truppen bevorzugt."
  
- tradition_nubian_warrior_queens_name:0 "Warrior Queens"
- tradition_nubian_warrior_queens_desc:0 "Those who live in Akavir well remember the great Warrior Queens who once forged kingdoms all over Tsakara. Our neighbors may insist on the superiority of males, but we track our bloodlines through our mothers, and the warcry of powerful Tsaesci women send any "men" running home."
+ tradition_nubian_warrior_queens_name:0 "Kriegerinnen"
+ tradition_nubian_warrior_queens_desc:0 "Die Personen die in Akavir leben gedenken an denn Kriergerinnen die einst Konigreiche in ganz Tsakara gründeten. Unsere Nachbarn mögen auf die Überlegenheit der Männder pochen, aber unsere Blutlinien gehen bis zu unseren Müttern, und der Kriegschrei der mächtigen Tsaesci Kriegerinnen schickt jeden "Mann" nach Hause."
```

Comment:
Jene die in Akavir leben gedenken noch immer den Kriegerinnen die einst Köngireiche in ganz Tsakara gründeten. Unsere Nachbarn mögen auf die Überlegenheit der Männer pochen, unsere Blutlinien verfolgen wir jedoch über unsere Mütter, und der Kriegschrei der mächtigen Tsaesci Kriegerinnen schickt jeden "Mann" schreiend nach Hause.